libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/ElementTextStyles.kt

```diff
@@ -25,6 +25,14 @@ import androidx.compose.ui.unit.sp
 // TODO Remove
 object ElementTextStyles {
 
+    val Button = TextStyle(
```

Comment:
I don't know if we want this or use some Composable instead

Reply:
We already have a custom `Button` composable, but its contents are 'anything goes' right now. We could modify it/create different a different version that takes just a `String` as content and builds the `Text` component inside with the proper style.

Reply:
Also, in the designs there are already some buttons with progress indicators along with the text, so maybe we do need to keep this `TextStyle` so we can add custom contents, or create a `ProgressButton` composable and also copy the style manually.